src/nni_manager/training_service/remote_machine/osCommands.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+'use strict';
+
+import { RemoteCommandResult } from "./remoteMachineData";
+
+abstract class OsCommands {
```

Comment:
`OSCommands will be better`

Reply:
as most naming convention guides, the abbreviation should follow Camel style also.